packages/theme-compatibility/src/base/editor.scss

```diff
@@ -1,9 +1,13 @@
-@import '~@wordpress/components/build-style/style.css';
-@import '~@wordpress/block-editor/build-style/style.css';
-@import '~@wordpress/block-library/build-style/style.css';
-@import '~@wordpress/block-library/build-style/editor.css';
-@import '~@wordpress/block-library/build-style/theme.css';
-@import '~@wordpress/format-library/build-style/style.css';
+@import 'base';
+
+// It seems we don't need those. commented and didn't notice any issues.
```

Comment:
I think they might all be included in `~isolated-block-editor/build-browser/core.css` anyway. Let's just remove these if they're not necessary.